src/core/border_router/routing_manager.hpp

```diff
@@ -198,6 +198,11 @@ class RoutingManager : public InstanceLocator
 
     static constexpr uint8_t kOmrPrefixLength    = OT_IP6_PREFIX_BITSIZE; // The length of an OMR prefix. In bits.
     static constexpr uint8_t kOnLinkPrefixLength = OT_IP6_PREFIX_BITSIZE; // The length of an On-link prefix. In bits.
+    static constexpr uint8_t kBrUlaPrefixLength  = 48;                    // The length of a BR ULA prefix. In bits.
+    static constexpr uint8_t kNat64PrefixLength  = 96;                    // The length of a NAT64 prefix. In bits.
+
+    static constexpr uint8_t kOmrPrefixSubnetId   = 1; // The subnet ID of a OMR prefix within the BR ULA prefix site.
+    static constexpr uint8_t kNat64PrefixSubnetId = 2; // The subnet ID of a NAT64 prefix within the BR ULA prefix site.
```

Comment:
Is the constant prefix subnet ID defined by Spec?

Reply:
The Spec doesn't specify how to allocate the subnet ID, but mentioned that the BR shall avoid conflicts of the allocated /64 prefixes. Here I use constant subnet IDs to avoid the conflicts for simplicity.

Reply:
SGTM